Rules/WAR/Scripts/WAR.as

```diff
@@ -723,7 +723,7 @@ shared class WarCore : RulesCore
 	void CheckTeamWon()
 	{
 		// can't lose/modify state if the match is not running
-		if (!rules.isMatchRunning()) { return; }
+		if (!rules.isMatchRunning() || rules.get_bool("tutorial")) { return; }
```

Comment:
```suggestion
		if (!rules.isMatchRunning()) { return; }
		if(rules.get_bool("tutorial"))
		{
			CBlob@[] halls;
			getBlobsByName("hall", @halls);
			if(halls[1].getTeamNum()!=0&&halls[1].getTeamNum()!=1) 
			{
				halls[1].server_setTeamNum(1);
				return;
			}
		}
```
This should make the last hall captured by the red team, making the game only end after capturing that hall as well.